packages/suite-desktop/src/modules/tor.ts

```diff
@@ -163,6 +163,22 @@ const load = async ({ mainWindow, store }: Dependencies) => {
         return { success: true };
     });
 
+    // Handle event emitted by request-interceptor module
+    let lastCircuitResetTime = 0;
+    mainThreadEmitter.on('module/reset-tor-circuits', event => {
```

Comment:
Maybe we could type this event?

Reply:
`mainThreadEmitter` is [already strongly typed](https://github.com/trezor/trezor-suite/blob/develop/packages/suite-desktop/src/modules/index.ts#L46-L49)
i somehow managed to put it into develop [while cherry-picking](https://github.com/trezor/trezor-suite/commit/c7e3a8f88a67e803696e6214a2c298004fdcf18f) :)